odoo_backup_sh/controllers/main.py

```diff
@@ -93,10 +93,17 @@ def update_cloud_params(cls, cloud_params, redirect, call_from):
         if not user_key:
             user_key = ''.join(random.choice(string.hexdigits) for _ in range(30))
             cls.set_config_values('options', {'odoo_backup_user_key': user_key})
-        cloud_params = requests.get(BACKUP_SERVICE_ENDPOINT + '/get_cloud_params', params={
-            'user_key': user_key,
-            'redirect': redirect,
-        }).json()
+
+        # If your website does not have a certificate and you are not sending sensitive data,
+        # you can set the verify parameter to False and requests will ignore SSL verification.
+        try:
+            cloud_params = requests.get(BACKUP_SERVICE_ENDPOINT + '/get_cloud_params', params={
+                'user_key': user_key,
+                'redirect': redirect,
+            }, verify=False).json()
+        except Exception:
+            _logger.exception('Failed to load cloud params')
```

Comment:
This try block doesn't prevent error raising later (cloud_params is undefined

Reply:
@yelizariev yes, but now we can open the Backups menu. Without this construction we will get exeption error therefore the Backups menu will not open